core/modules/simpletest/simpletest.module

```diff
@@ -411,8 +428,17 @@ function simpletest_test_get_all() {
           }
         }
         if (!empty($error)) {
-          watchdog('simpletest', $error, array('%uri' => $file->uri), WATCHDOG_WARNING, l(t('See the Testing Framework documentation.'), 'https://docs.backdropcms.org/documentation/testing-framework'));
-          continue;
+          if (backdrop_is_cli()) {
+            // If running in CLI, then we're probably about to run some
+            // automated tests, so we need a hard fail.
+            print format_string($error, $variables) . "\n";
+            exit(1);
+          }
+          else {
+            backdrop_set_message(format_string($error, $variables), 'error');
```

Comment:
If this is shown in the UI, we should be using t() here, shouldn't we?

Reply:
No, because `$error` has already been translates (for the reasons explained in the code comments just removed per the previous comment).